wrangler-core/src/main/java/io/cdap/directives/column/SplitToColumns.java

```diff
@@ -99,5 +102,26 @@ public void destroy() {
     }
     return results;
   }
+
+  @Override
+  public Mutation lineage() {
+    return Mutation.builder()
+      .readable("Split the column '%s' with regex '%s'", column, regex)
+      .relation(
+        column,
+        Many.columns(
+          column,
+          String.format("%s_%d", column, 1),
+          String.format("%s_%d", column, 2),
```

Comment:
We are hard coding the columns here, should we use ALL relation in this case?

Reply:
No, I would rather use fixed set. If there is nothing beyond this for a column, it would be ignored anyway.